### PR TITLE
[ticket/17235] missing autocomplete for username

### PR DESCRIPTION
**Pull request by Prosk8er**
PHPBB3-17235

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/master/coding-guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-17235

### CODE REVIEW

File: phpBB/styles/prosilver/template/login_forum.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2 class="login-title">{L_LOGIN}</h2>
 
 			<dl>
 				<dt><label for="password">{L_PASSWORD}{L_COLON}</label></dt>
-				<dd><input type="password" tabindex="1" id="password" name="password" size="25" class="inputbox narrow" autocomplete="off" /></dd>
+				<dd><input type="password" tabindex="1" id="password" name="password" size="25" class="inputbox narrow" autocomplete="current-password" /></dd>
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/login_forum.html:25]: This is for forum passwords and not the user password

File: phpBB/styles/prosilver/template/ucp_register.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h2>{SITENAME} - {L_REGISTRATION}</h2>
 	<!-- EVENT ucp_register_credentials_before -->
 	<dl>
 		<dt><label for="username">{L_USERNAME}{L_COLON}</label><br /><span>{L_USERNAME_EXPLAIN}</span></dt>
-		<dd><input type="text" tabindex="1" name="username" id="username" size="25" value="{USERNAME}" class="inputbox autowidth" title="{L_USERNAME}" /></dd>
+		<dd><input type="text" tabindex="1" name="username" id="username" size="25" value="{USERNAME}" class="inputbox autowidth" title="{L_USERNAME}" autocomplete="username" /></dd>
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/ucp_register.html:41]: I don't think it makes sense to have autocomplete while registering

**Review by marc1706 — CHANGES_REQUESTED** [review]

File: phpBB/styles/prosilver/template/login_forum.html (hunk 1 above)
Comment (Prosk8er) [reply phpBB/styles/prosilver/template/login_forum.html:25]: ooops thanks i'll fix that

File: phpBB/styles/prosilver/template/ucp_register.html (hunk 2 above)
Comment (Prosk8er) [reply phpBB/styles/prosilver/template/ucp_register.html:41]: ok i'll switch it to off

**Review by marc1706 — APPROVED** [review]

**Comment by marc1706** [comment]
@Prosk8er Thanks for the PR :+1:

**Comment by Prosk8er** [comment]
> @Prosk8er Thanks for the PR 👍

your welcome glad to help when i can

**Pull request merged**